doc/classes/ARVRServer.xml

```diff
@@ -26,6 +44,15 @@
 				You should call this method after a few seconds have passed. For instance, when the user requests a realignment of the display holding a designated button on a controller for a short period of time, or when implementing a teleport mechanism.
 			</description>
 		</method>
+		<method name="clear_primary_interface_if">
```

Comment:
That seems a bit weird as API. I'm not familiar with using `ARVRInterface`s but I assume there's only one primary interface? If so, why is it needed to pass the object?

I'd expect something more like this:
```gdscript
if ARVRServer.get_primary_interface() == my_interface:
    ARVRServer.clear_primary_interface()
```

But again, not familiar with this API so it might be best as is.

Reply:
It's enforced in the ARVRServer but in hindsight, the code suggestion you make here is way better then what I introduced 3 years ago. Worth a change in master :)